mixer/testdata/config/accesslog.yaml

```diff
@@ -16,16 +16,36 @@ spec:
   timestamp: request.time
   variables:
     sourceIp: source.ip | ip("0.0.0.0")
-    destinationService: destination.service | "unknown"
```

Comment:
why remove destination service?

Reply:
`destination.service` attribute is deprecated. This was replaced with `destinationServiceHost` below (as `destination.service.host` is the replacement attribute.

Reply:
Can we add destination_name: destination.name | "unknown" like it's there in metrics.yaml ?